web/src/pages/AdminPage/UserEditor.tsx

```diff
@@ -0,0 +1,77 @@
+import {
+    Button,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogContentText,
+    DialogTitle,
+    TextField,
+} from '@mui/material'
+import * as React from 'react'
+import { useEffect, useState } from 'react'
+import { User, UserUpdateRequest } from '../../model/user'
+
+interface UserEditorProps {
+    user: User | null
```

Comment:
Why is it nullable? When do we want to edit a non-existing user?

Reply:
i left it like that to later add a button like `add user` at the bottom of the list, i can remove it if you want